guide-us-english.tex

```diff
@@ -164,26 +163,27 @@ \subsection{Values-Driven Culture}
 this must occur in an environment based on respect for the individuals
 doing the work, without whom nothing can be created.
 
-Scrum@Scale helps organizations thrive by supporting both a
-servant-leadership and intent-based leadership model,\footnote{Marquet, L
-David, Turn the Ship Around!: How to Create Leadership at Every Level,
-Greenleaf Book Group, 2012} which fosters a positive environment for
+Scrum@Scale helps organizations thrive by supporting a
+transformational leadership model which fosters a positive environment for
 working at a sustainable pace and putting commitment to deliver
 customer-facing value at the forefront of our efforts.
 
 \subsection{Getting Started with Scrum@Scale}
 When implementing large networks of teams, it is critical to develop a
 scalable \textbf{Reference Model} for a small set of teams. Any
 deficiencies in a Scrum implementation will be magnified when multiple
-teams are deployed.
+teams are deployed. Many of the initial scaling problems will be organizational 
+policies and procedures or development practices that block high performance and frustrate 
+employees. 
```

Comment:
"employee" has a different tone than the rest of the guide, can we say "team members" or "teams"